src/app/util/ember-compatibility-functions.cpp

```diff
@@ -516,6 +516,43 @@ Protocols::InteractionModel::Status UnsupportedAttributeStatus(const ConcreteAtt
 
 } // anonymous namespace
 
+template <typename T>
+CHIP_ERROR EncodeDefaultForUnfoundManditoryAttribute(const ConcreteReadAttributePath & aPath,
```

Comment:
Other things that need to happen, in addition to this, for this to actually be spec-compliant:

1. We need to list the attribute in `AttributeList`.
2. We need to make wildcard expansion see the attribute.

Reply:
Sorry for all the spam, but as I move forward I am answering some of my on questions:

>Please let me know if I am missing something here and actually there are more wildcards I need to account for.

I can also wildcard the cluster ID: `chip-tool any read-by-id 0xffffffff 0xfffd 12344321 0`. If there is any other wildcards I need to account for please let me know.

Reply:
You can wildcard the attribute too, as `0xffffffff`.

The wildcard expansion bits live in `AttributePathExpandIterator.cpp`.